certbot/compat/misc.py

```diff
@@ -142,8 +146,13 @@ def get_default_folder(folder_type):
 
     """
     if os.name != 'nt':
-        # Linux specific
-        return LINUX_DEFAULT_FOLDERS[folder_type]
+        # Unix-like
+        if sys.platform.startswith('freebsd') or sys.platform.startswith('dragonfly'):
+            # FreeBSD specific
+            return FREEBSD_DEFAULT_FOLDERS[folder_type]
+        else:
```

Comment:
The failure in the build is due to a linting error here. The `else` is not necessary, since we return in L152 if condition on L150 is `True`. So L154 and L155 can be executed only if the condition is `False`.

Reply:
Should be fixed in d0a50ef92f6085f14a36c8fc076f2caa7af711c4.